magma_access_gateway_post_install/__init__.py

```diff
@@ -14,27 +14,25 @@
 from systemd.journal import JournalHandler  # type: ignore[import]
 
 from .agw_post_install import AGWPostInstallChecks
-from .agw_post_install_errors import PostInstallError
 
 logger = logging.getLogger(__name__)
-handler = JournalHandler()
-handler.setFormatter(logging.Formatter("Magma AGW Post-Install: %(message)s"))
-logger.addHandler(handler)
 logger.setLevel(logging.INFO)
-logger.addHandler(logging.StreamHandler(sys.stdout))
+journal_handler = JournalHandler()
+journal_handler.setFormatter(logging.Formatter("Magma AGW Post-Install: %(message)s"))
+logger.addHandler(journal_handler)
+stdout_handler = logging.StreamHandler(sys.stdout)
+stdout_handler.setFormatter(logging.Formatter("Magma AGW Post-Install: %(message)s"))
```

Comment:
Do we need the "Magma AGW Post-Install" section? when you run magma-access-gateway.post-install you already know you're in the context of post-installing magma.

Reply:
I added it here to be consistent with installation process (see my answer above for the reason why I've added it to the installer).